### ISSUE TITLE
Multi-line output (pretty print) empty object declaration screws up formatting

### ISSUE THREAD

**Issue by drwestco** [trullock/NUglify#239]
NUglifyApp, v1.13.6.

Input:
```
(function ()
{
    try
    {
        var a = {};
    }
    catch (ex)
    {
    }
})();
```

NUglifyApp -pretty:
```
(function()
{
    try
    {
        var a = {  }
}
catch(ex) {}
})()
```

Note two unnecessary spaces inside the object literal assignment.
Note indentation level lost after the object literal assignment.

**Comment by trullock** [comment]
the object whitespace is probably a matter of taste, but the indentation is definitely a bug. Will investigate

**Comment by drwestco** [comment]
Yeah, the object whitespace is understandable. It's an artifact of a single-line object declaration with space between braces and object properties. In the case of no properties, having two spaces inside is a little odd.

**Comment by trullock** [comment]
Fixed in 1.13.7
